docker/install_node.sh

```diff
@@ -14,10 +14,17 @@
 # limitations under the License.
 
 #!/bin/bash
+# Check if node is installed. If it is, skip installation.
+if [ -d "../oppia_tools/node-16.13.0" ]; then
+    echo "Node.js is already installed. Skipping the installation."
+    exit 0
+fi
+
+# Install node.js if it is not installed.
 OS_NAME=$(uname)
 echo "Installing Node.js..."
 
-if [ "$OS_NAME" = "Windows" ]; then
+if [ "$OS_NAME" = "Windows" ] || [ $(uname -o) = "Msys" ]; then  # uname in Windows 11 also adds version, so uname -o is used.
```

Comment:
Please modify the comment to be similar to: We are doing ... because ...
Also, please move the comment to the previous line.

Reply:
done!

Reply:
@jayam04 Sorry for not being clear. I'm asking you to write a comment explaining why we should check for "Msys" also for windows systems? when we're using `"$OS_NAME" = "Windows"` to do the same task?